sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/jedis/JedisRedisCheckpointStore.java

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.messaging.eventhubs.checkpointstore.jedis;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
+import com.azure.messaging.eventhubs.CheckpointStore;
+import com.azure.messaging.eventhubs.models.Checkpoint;
+import com.azure.messaging.eventhubs.models.PartitionOwnership;
+import reactor.core.Exceptions;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.Jedis;
+
+/**
+ * Implementation of {@link CheckpointStore} that uses Azure Redis Cache, specifically Jedis.
+ */
+public class JedisRedisCheckpointStore implements CheckpointStore {
+    private static final ClientLogger LOGGER = new ClientLogger(JedisRedisCheckpointStore.class);
+    private final JedisPool jedisPool;
+    private final JacksonAdapter jacksonAdapter = new JacksonAdapter();
```

Comment:
We don't use the Jackson adapter as-is. We have an ObjectSerializer we use on top of this.  See:  https://docs.microsoft.com/en-us/java/api/com.azure.core.util.serializer.jacksonadapter?view=azure-java-stable

Reply:
Thank you, I updated it to use ObjectSerializer!